server/channels/db/migrations/migrations.list

```diff
@@ -543,3 +543,6 @@ channels/db/migrations/postgres/000136_create_attribute_view.down.sql
 channels/db/migrations/postgres/000136_create_attribute_view.up.sql
 channels/db/migrations/postgres/000137_update_attribute_view.down.sql
 channels/db/migrations/postgres/000137_update_attribute_view.up.sql
+channels/db/migrations/postgres/000138_add_remoteid_channelid_to_post_acknowledgements.down.sql
```

Comment:
The entry for MySQL is not there. I hope you are not creating this line manually. This is meant to be auto-generated with `make new-migration name=<>`.

Reply:
updated